Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -134,6 +134,9 @@ public protocol AppDelegateViewModelOutputs {
   /// Emits when the root view controller should navigate to the creator dashboard.
   var goToDiscovery: Signal<DiscoveryParams?, Never> { get }
 
+  /// Emits when the root view controller should present the Landing Page for new users.
+  var goToLandingPage: Signal<LandingPageViewController, Never> { get }
```

Comment:
This can just be a `Void` or `()` signal and we can instantiate the VC outside of the VM.